src/ProfileEndpoints.jl

```diff
@@ -246,11 +271,39 @@ end
 
 end  # if isdefined
 
+###
+### Debug super-endpoint
+###
+
+function debug_profile_endpoint_with_stage_path(stage_path = nothing)
+    stage_path = stage_path === nothing ? tempdir() : stage_path
+    function debug_profile_endpoint(req::HTTP.Request)
+        uri = HTTP.URI(req.target)
+        qp = HTTP.queryparams(uri)
+        if isempty(qp)
+            @info "TODO: interactive HTML input page"
+            return HTTP.Response(400, cpu_profile_error_message())
+        end
+        req_type = get(qp, "type", "profile")
+        @info "Debugging endpoint hit with type: $req_type"
+        if req_type == "profile"
```

Comment:
I think it was a mistake that we originally used `profile` to refer to cpu profiles.

Please use `cpu_profile` everywhere in the interface

Reply:
SGTM.